### PR TITLE
Throw when trying to save insight with empty filters

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

- Somehow, sometimes we save insights with empty filters. This has caused data loss for some clients already. Instead of continuing this practice, let's fail loudly and debug.

## How did you test this code?

- Made a test.

### CODE REVIEW

**Comment by mariusandra** [comment]
Weirdly, every other reload gives an empty graph:



This PR at least plugs the hole that happens if this is now (automatically?) saved.

File: frontend/src/scenes/insights/insightLogic.tsx [hunk 1]
```diff
@@ -159,6 +175,10 @@ export const insightLogic = kea<insightLogicType>({
                         return { ...values.insight, ...metadata }
                     }
 
+                    if (metadata.filters) {
+                        throw new Error('Can not update "filters" via setInsightMetadata')
```
Comment (Twixes) [inline frontend/src/scenes/insights/insightLogic.tsx:179]: Should this have `Sentry.captureException` too?

**Comment by mariusandra** [comment]
Managed to get this to a state where all tests pass 😌

File: cypress/integration/trends.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ describe('Trends', () => {
         cy.location('pathname').should('include', '/edit')
     })
 
-    it('Can load a graph from a URL directly', () => {
+    it.skip('Can load a graph from a URL directly', () => {
```
Comment (Twixes) [inline cypress/integration/trends.js:9]: This is A LOT of Volkswagen, are you sure about this?

Comment (mariusandra) [reply cypress/integration/trends.js]: Trying The Toyota Way this time...

**Review by Twixes — APPROVED** [review]
Let's give this a go

**Pull request merged**